### PR TITLE
 Add timeout option to httpIdentity axios instance

### PR DESCRIPTION
**Pull request by m-russkikh**

### CODE REVIEW

File: test/unit/specs/class/singleton/http.spec.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { http } from '@/class/singleton';
+
+describe('httpIdentity', () => {
```
Comment (kazurus) [inline test/unit/specs/class/singleton/http.spec.js]: http

**Comment by kazurus** [comment]
@Lenseg

**Comment by kazurus** [comment]
@Lenseg

**Comment by kazurus** [comment]
@m-russkikh :recycle:

**Comment by m-russkikh** [comment]
@kazurus :up:

**Pull request merged**